packages/riipen-ui/src/components/Grid.jsx

```diff
@@ -45,14 +69,63 @@ class Grid extends React.Component {
     /**
      * Defines the space between grid items in the grid.
      */
-    spacing: PropTypes.oneOf(SPACINGS)
+    spacing: PropTypes.oneOf(SPACINGS),
+
+    /**
+     * The function to callback to when the grid changes sizes
+     * The callback includes the current size of the grid.
+     */
+    sizeChange: PropTypes.func
```

Comment:
I would potentially rename this one. All of our callback function props usually start with `on`. So like `onChange` or something. Maybe call this one `onResize` or `onSizeChange`?